src/components/Form/Form.js

```diff
@@ -1,1855 +1,168 @@
+            setSubmittedData(formData);
+            setShowReceipt(true);
+        }
+        return errors;
+    }, [onSubmit]);
+
+    console.log(base);
+    const renderStep = useCallback((index, {label, name, fields}, errors, submitting) => (
+        <Step key={label}>
+            <StepLabel className={classes.stepLabel}>{label}</StepLabel>
+            <StepContent>
+                {fields.map((field) => React.cloneElement(field,
+                    {
+                        "SelectDisplayProps": {
+                            "data-cy": `${name}-${field.props.name}-select`,
+                        },
+                        "data-cy": `${name}-${field.props.name}`,
+                        "inputProps": {
+                            "data-cy": `${name}-${field.props.name}-input`,
+                        },
+                        "name": `${name}.${field.props.name}`,
+                        "onBlur": (...params) => {
+                            console.log(field.onChange)
+                            console.log("BLUUR")
+                        },
+                    }))}
+                {index > 0 && index < sections.length &&
+                    <Button data-cy="backButton" onClick={handleBack}>
+                        Back
+                    </Button>}
+                {index < sections.length - 1 &&
+                    <Button data-cy="nextButton"
+                        disabled={Boolean(errors[name])} onClick={handleNext}>
+                        Next
+                    </Button>}
+                {index === sections.length - 1 &&
+                    <Button data-cy="submitButton"
+                        disabled={Boolean(errors[name]) || submitting}
+                        type="submit">
+                        {submitting ? "Submitting" : "Submit"}
+                    </Button>}
+            </StepContent>
+        </Step>
+    ), [classes.stepLabel, sections.length, handleBack, handleNext]);
+
+    const render = useCallback(({handleSubmit, errors, submitError, submitting}) => (
+        <form noValidate onSubmit={handleSubmit}>
+            <Stepper activeStep={activeStep} orientation="vertical">
+                {sections.map((section, index) =>
+                    renderStep(index, section, errors, submitting))}
+            </Stepper>
+            {submitError &&
+                <div className="error">
+                    An error occured while submitting. Try again.
+                </div>}
+            <Debug />
+        </form>
+    ), [activeStep, renderStep, sections]);
+
+    return (
+        // <Paper className={`registration-form ${classes.root}`}>
+        //     <BackButton />
+        <div className={`registration-form ${classes.root}`}>
+            <Typography align="left" className="heading" data-cy="formTitle"
+                variant="h3">
+                {title}
+            </Typography>
+            {showReceipt ?
+				<Receipt format={base} formData={submittedData} /> :
```

Comment:
check if there's a receipt prop passed into Form.js. If there isn't, then default to this current Receipt

Reply:
Receipt prop defaults to FormReceipt if it's not provided, is this what you want?